libs/vkd3d/memory.c

```diff
@@ -1325,6 +1337,10 @@ static HRESULT vkd3d_memory_allocation_init(struct vkd3d_memory_allocation *allo
         return hr;
     }
 
+    if ((type_flags & VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT) &&
```

Comment:
Shouldn't this be the type we actually end up allocating? This seems very wrong.

Reply:
Dropped, doing the flush unconditionally now.